app/models/batch/updater.rb

```diff
@@ -19,7 +19,7 @@ def update!
     private
 
     def asset_params
-      { study: study, project: project, workflow_id: workflow, pipeline_destination_id: pipeline_destination,
+      { study: study, project: project, workflow_id: workflow.id, pipeline_destination_id: pipeline_destination,
```

Comment:
I'd suggest being consistent and just using the object association across the board here, although rails usually does coerce a record to an id when passed directly to an id attribute (As it does for the other parameters in this, I'm not sure why you needed to be explicit for workflow)